go-runtime/schema/analyzers/initialize.go

```diff
@@ -0,0 +1,79 @@
+package analyzers
+
+import (
+	"errors"
+	"go/token"
+	"go/types"
+	"reflect"
+
+	"github.com/TBD54566975/golang-tools/go/analysis"
+	"github.com/TBD54566975/golang-tools/go/analysis/passes/inspect"
+	"github.com/TBD54566975/golang-tools/go/packages"
+)
+
+// Initializer prepares data prior to the schema extractor runs, e.g. loads FTL types for reference by other
+// analyzers.
+var Initializer = &analysis.Analyzer{
+	Name:             "ftlTypeLoader",
+	Doc:              "extracts type aliases to the module schema",
+	Run:              initialize,
+	Requires:         []*analysis.Analyzer{inspect.Analyzer},
+	ResultType:       reflect.TypeFor[initializeResult](),
+	RunDespiteErrors: true,
+}
+
+type initializeResult struct {
+	ftlTypes map[string]*types.Interface
+}
+
+func (r initializeResult) isFtlErrorType(typ types.Type) bool {
```

Comment:
nittiest of nits: `isFTLErrorType` for go [style](https://google.github.io/styleguide/go/decisions#initialisms)